code/_onclick/other_mobs.dm

```diff
@@ -422,21 +422,21 @@
 								V.dropItemToGround(picked)
 								put_in_active_hand(picked)						
 								to_chat(src, span_green("I stole [picked]!"))
-								V.log_message("has had \the [picked] stolen by [key_name(U)]", LOG_ATTACK, color="black")
-								U.log_message("has stolen \the [picked] from [key_name(V)]", LOG_ATTACK, color="black")
+								V.log_message("has had \the [picked] stolen by [key_name(U)]", LOG_ATTACK)
+								U.log_message("has stolen \the [picked] from [key_name(V)]", LOG_ATTACK)
```

Comment:
I have no fucking idea what V and U are and this proc is massive and full of this. Definitely need to go around and change those variable names, but that's very out of scope for this PR- just something I wanted to note.